flaskapp/view_functions/ivrflow.py

```diff
@@ -335,7 +348,26 @@ def get_txt_from_url(url):
 
     x = {"text1": text[0:15002], "text2": text[15002:30000]}
     return (jsonify(x))
+
+
 def get_term_cond():
     return get_txt_from_url('https://www.iubenda.com/terms-and-conditions/86762295')
+
+
 def get_privacy():
-    return get_txt_from_url('https://www.iubenda.com/privacy-policy/86762295/full-legal')
\ No newline at end of file
+    return get_txt_from_url('https://www.iubenda.com/privacy-policy/86762295/full-legal')
```

Comment:
Such texts (licence, policy, legal etc) not too heavy, so I would recommend to preload them when the program starts; This significantly reduces the response time when such  `get_privacy` will be called. Reading url, at least in this implementation, is blocking operation, so the user will wait while we will access `iubenda.com` (what if iubenda.com will not respond?)...In case of preloading, we can much smoothly handle such things as inaccessibility of `iubenda.com`...

Reply:
Please tell me where can I do preload and how? I am new in this project . And this task I had not completed.

Reply:
@scidam this is a good point. let's fix this as a separate issue, I'm sure this is slightly related to get_profile function.